### DISCUSSION TITLE
About the fit property value of the Image component

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#2244] [Q&A] [answered]
## Description

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

![image](https://user-images.githubusercontent.com/27342882/186901937-aba2e47e-eb64-4eee-b21d-d0c296423daa.png)

When referring to MDN, several properties of object-fit exist. Is the absence of values ​​other than 'contain' and 'cover' intentional?

If it's not intentional, I'll post a PR.

https://mantine.dev/core/image/?t=props

**Comment by rtivital** [comment, marked as answer]
Yeah, I think we can change it to `React.CSSProperties['objectFit']`, feel free to send a PR
